third_party/kubevela/kubevela/apis/common/deepcopy_generated.go

```diff
@@ -0,0 +1,272 @@
+package common
+
+import (
+	v1 "k8s.io/api/core/v1"
+	runtime "k8s.io/apimachinery/pkg/runtime"
+
+	"kusionstack.io/kusion/third_party/kubevela/workflow/api/v1alpha1"
+)
+
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
```

Comment:
According to my understanding, we only need the kubevela `Application` definition to simplify our watch command. Do we need all these autogenerated DeepCopy codes and where will we use them?

Reply:
Manifest(YAML) must be converted to Application(runtime.Object), so that health checker and printer know the field path.

Reply:
ok, I got it. DeepCopyObject() is an interface method of runtime.Object

Reply:
yep